format/avro/decoders/bytes.go

```diff
@@ -0,0 +1,18 @@
+package decoders
+
+import (
+	"github.com/wader/fq/format/avro/schema"
+	"github.com/wader/fq/pkg/decode"
+)
+
+type BytesCodec struct{}
+
+func decodeBytesFn(schema schema.SimplifiedSchema) (func(string, *decode.D), error) {
+	//bytes are encoded as a long followed by that many bytes of data.
+	return func(name string, d *decode.D) {
+		// What if its a record with a field called name_len?
+		// using a struct is probably a better idea. But it makes it less usable
```

Comment:
Other codec types are more than just one value so maybe it make sense to have as struct? But it's a fine line, i've usually decoded length prefix strings and null terminated string as one value but raw bytes i'm not sure how i would do it.